torchbenchmark/__init__.py

```diff
@@ -14,6 +15,29 @@
 model_dir = 'models'
 install_file = 'install.py'
 
+# Enum class to hold all the available domain names
```

Comment:
when i suggested to move to a new file, it's not because this is 'wrong' but just to make the import cleaner.  Moving it only helps if you also make the other changes I suggested, since if you made my changes and left them here you'd have 
from torchbenchmark import ComputerVision
which is a little less clean than
from torchbenchmark.domains import ComputerVision

and the .domains comes from putting them in a file domains.py inside torchbenchmark.